scripts/deploy-pr.sh

```diff
@@ -63,30 +68,20 @@ fi
 branchName=$(gh pr view --json headRefName | jq -r .headRefName)
 prNumber=$(gh pr view --json number | jq -r .number)
 
-if $skipBuild; then
-	#check if the image exists
-	foundTag=$(curl -fsSL https://github.com/coder/coder/pkgs/container/coder-preview | grep -o "$prNumber" | head -n 1) || true
-	echo "foundTag is: '${foundTag}'"
-	if [[ -z "${foundTag}" ]]; then
-		echo "Image not found"
-		echo "${prNumber} tag not found in ghcr.io/coder/coder-preview"
-		echo "Please remove --skip-build and try again"
-		exit 1
-	fi
-fi
-
 if $dryRun; then
```

Comment:
You should use actual comparisons here because right now it's running whatever program is stored in the `$dryRun` variable. On linux, `true` and `false` exist so this works, but you shouldn't rely on this. Use `[[ "$dryRun" == "true" ]]` instead or something similar